frontend/packages/topology/src/components/quick-search/QuickSearchModalBody.tsx

```diff
@@ -77,12 +87,16 @@ const QuickSearchModalBody: React.FC<QuickSearchModalBodyProps> = ({
   );
 
   const onEnter = React.useCallback(() => {
-    const viewAllLink = document.getElementById('viewAll');
+    const devCatalog = document.getElementById('catalog');
+    const quickStartCatalog = document.getElementById('quickstarts');
```

Comment:
I see now where the IDs are being used.
I think the enter key should only work when focused on the text input. The only other items to focus are the view all links and the CTA buttons. Both of which can be invoked using the keyboard already.
If your listener for the enter key is on the text input only, then you won't have to perform these checks.